skt/executable.py

```diff
@@ -793,6 +793,12 @@ def load_config(args):
     for idx, statefile_path in enumerate(cfg.get('states') or []):
         cfg['states'][idx] = full_path(statefile_path)
 
+    if cfg.get('junit'):
+        try:
+            os.mkdir(cfg.get('junit'))
```

Comment:
Could you wrap this in `full_path()` so we get an absolute path to the directory?